packages/api/src/lib/granule-delete.ts

```diff
@@ -0,0 +1,100 @@
+import Knex from 'knex';
+import pMap from 'p-map';
+
+import { deleteS3Object } from '@cumulus/aws-client/S3';
+import {
+  GranulePgModel,
```

Comment:
def a nit but you can alphabetize

```suggestion
  FilePgModel,
  GranulePgModel,
  PostgresFileRecord,
  PostgresGranuleRecord,
```

apologies that this suggestion doesn't work super neatly